langchain/llms/replicate.py

```diff
@@ -109,8 +112,20 @@ def _call(
             key=lambda item: item[1].get("x-order", 0),
         )
         first_input_name = input_properties[0][0]
-
         inputs = {first_input_name: prompt, **self.input}
-        iterator = replicate_python.run(self.model, input={**inputs, **kwargs})
 
-        return "".join([output for output in iterator])
+        if self.streaming:
+            prediction = replicate_python.predictions.create(
```

Comment:
why do we need to use a different method than replicate_python.run? doesn't that already stream outputs when possible (per https://replicate.com/docs/get-started/python)

Reply:
Great catch! I'll see if I can just use the iterator returned by .run(...). Thank you!

Reply:
Fixed... much cleaner now, thanks.